src/hooks/use-mood-block.hook.ts

```diff
@@ -40,16 +50,21 @@ function useMoodBlock(): MoodBlock {
   const decrementYear = () => {
     setYear((prevYear) => prevYear - 1)
   }
-  const getDays = (myYear: number, myMonth: number) => {
-    return Array.from(
-      { length: new Date(myYear, myMonth + 1, 0).getDate() },
-      (_, i) => new Date(myYear, myMonth, i + 1)
-    )
+  const getDays = (myYear: number, myMonth: number): Date[] => {
+    const daysInMonth = new Date(myYear, myMonth + 1, 0).getDate()
+    const result: Date[] = new Array(daysInMonth)
+
+    for (let index = 0; index < daysInMonth; index++) {
+      result[index] = new Date(myYear, myMonth, index + 1)
+    }
+
+    return result
   }
 
   const days = getDays(year, month)
```

Comment:
so, i guess we don't need this function, we can create constant days with useMemo and put all needed functionality